### PR TITLE
Release 3.2.0-a1

### PR DESCRIPTION
**Pull request by Nicofuma**

### CODE REVIEW

File: src/Phpbb/Area51Bundle/Controller/DefaultController.php [hunk 1]
```diff
@@ -128,14 +128,14 @@ public function docsRedirectAction($path)
...
     {
         // Make this false when the most recent release is not an RC/Alpha/Beta
-        $latestDevelopment = false;
+        $latestDevelopment = true;
 
-        $previousVersions = array('3.0.12', '3.0.13', '3.0.13-PL1', '3.1.0', '3.1.1', '3.1.2', '3.1.3', '3.1.4', '3.1.5');
-        $currentVersion = '3.1.6-RC1';
-        $mainPackageSha = '2847bbf024f40d95c14feba3f4143cab2e410d967e8c3360665bab56ff25785b';
-        $subsilverSha = '5e461052aff673b1a929979ed9c184fbda8e51a79f3383b437cf4ce0898dcc26';
+        $previousVersions = array('3.0.14', '3.1.0', '3.1.1', '3.1.2', '3.1.3', '3.1.4', '3.1.5', '3.1.6');
```
Comment (michaelcullum) [inline src/Phpbb/Area51Bundle/Controller/DefaultController.php]: Between x.y releases we don't offer upgrade packages from all of those in the previous branch and the recommended method for updating is file replacement?

**Comment by bantu** [comment]
3.1.0-a1?

File: src/Phpbb/Area51Bundle/Controller/DefaultController.php (hunk 1 above)
Comment (Nicofuma) [inline src/Phpbb/Area51Bundle/Controller/DefaultController.php]: we need them anyway, one of the major change is the new installer/updater. We have to test it

**Comment by Nicofuma** [comment]
Just a typo in the title :) Thanks @michaelcullum for the rename ^^

**Pull request merged**